### ISSUE TITLE
Troubleshooting your WordPress site: Plugin and theme conflicts  

### ISSUE THREAD

**Issue by digitalchild** [WordPress/Learn#2311]
# Details

- Content type (Online Workshop, Lesson, Course, Tutorial, or Lesson Plan): Lesson
- Content title: Troubleshooting your WordPress site
- Topic description: 
White screen of death
Troubleshooting methods
Deactivating plugins
Health Check and Troubleshooting Plugin

- Audience (User, Developer, Designer, Contributor, etc.): User
- Experience Level (Beginner, Intermediate, Advanced, Any): Advanced 

# Learning Objectives

# Related Resources and Other Notes

**Comment by digitalchild** [comment]
//lessons

**Comment by ironnysh** [comment]
[Common WordPress errors](https://developer.wordpress.org/advanced-administration/wordpress/common-errors/):
> This page lists the most common WordPress errors experienced by WordPress users, and provides a starting point for fixing them.

**Comment by kspilarski** [comment]
I will create the script for this lesson. :)

**Comment by westnz** [comment]
Thank you for the work on the [script](https://docs.google.com/document/d/1aYq6OBRFKQcxbo6g0cCytdZDi7cxOf5FTu5Aae0rR38/edit) @kspilarski.

**Comment by ervanyuff** [comment]
Hi team, I would like to be involved in editing this video 📹

**Comment by westnz** [comment]
Kudos to @kspilarski for completing the script, voice & screen recordings. ⭐ 

Everything you need for the video is in this file @ervanyuff : https://drive.google.com/drive/folders/1Svj81F-wmX1vR2BJe8CWbpzBoz9Z2EJb?usp=drive_link

**Comment by westnz** [comment]
Thumbnail
<img width="1280" alt="Block text (5)" src="https://github.com/user-attachments/assets/de4f7a81-5e53-40ad-8342-a6004a6dc7a8">

**Comment by ervanyuff** [comment]
Hi team, here's [the video](https://drive.google.com/file/d/1N6in5YPYS_Rkm7DOnCq5iDlICufcKkHt/view?usp=sharing) Please feel free to provide any feedback. Thank you 🙏

**Comment by mkastler** [comment]
## Tutorial/Lessons Review Checklist
Please tick all items you've confirmed:
- [ ] Learning outcomes/objectives are clear.
- [X] Technical concepts introduced in the content are accurate.
- [X] The speed of demonstrations are easy to follow.
- [ ] The narration audio matches what is shown visually.
- [ ] Spelling and grammar are correct.
- [X] Sound quality is consistent throughout the video.
- [ ] [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/) are being followed.
- [ ] Media assets are all in the public domain ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)).

Leave an additional comment below with feedback. You can also share what you liked about this Tutorial.
---------
0:49 - Learning is spelled wrong (leaning)
1:26 - Implication is we will learn to at least identify, if not resolve, all common causes.  However, the only cause we are shown how to investigate is plugin conflict and theme issue.  A definition of each cause at least would be useful.
1:40 - Verbally numbered steps are not reflected as such on the video
1:54 - Says "systemically" but bullet point is "systematically".  Systematically is the appropriate word here imho.
2:04 - May want to suggest option for creating a staging or development site? 
2:30 - Doesn't show how to access error logs or debug files without a plugin, or what will be revealed by the debug files. How do you parse/interpret these files to determine what the cause of your issue is?
3:30 - Poor grammar on first sentence, as the "since" is left hanging without resolution/definition.  Would also suggest "The Rollback feature ... " or updating text to match verbal comments.
4:08 - Video shows renaming of individual plugin folders, not the top level plugins directory, even though that is what is suggested verbally
4:35 - Unclear instructions, due to issue mentioned above
5:30 - Verbally numbered steps are not reflected as such on the video
6:45 - Conclusion does not reflect the initial learning goals.  Should reiterate that you are being equipped to troubleshoot as described at 0:22.  Ideally should recap in short form what was discussed.
7:00 - Poor grammar on last slide, should say "links to troubleshooting support documentation **are** in the description"

GENERAL:  I feel like the presentation suffers from some lack of organization or focus ...  It might be useful to describe what the steps will be in advance - as in "first, we will see how to deactivate all your plugins, and then reactivate them one by one to see which may be the issue" and "here, I am going to show you how to deactivate your active theme and use a default one, whether or not you can access the dashboard."  This type of foreshadowing can help comprehension, and make it easier for people that know a little bit of what is going on and just need a pointer in the right direction. 

Overall this is a great topic and promises to be super useful!

**Comment by westnz** [comment]
Thank you for the valuable feedback @mkastler

**Comment by lintonhale** [comment]
## Tutorial/Lessons Review Checklist
Please tick all items you've confirmed:
- [ ] Learning outcomes/objectives are clear.
- [ ] Technical concepts introduced in the content are accurate.
- [ ] The speed of demonstrations are easy to follow.
- [ ] The narration audio matches what is shown visually.
- [ ] Spelling and grammar are correct.
- [ ] Sound quality is consistent throughout the video.
- [ ] [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/) are being followed.
- [ ] Media assets are all in the public domain ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)).

Leave an additional comment below with feedback. You can also share what you liked about this Tutorial.

**Comment by cat-og** [comment]
## Tutorial/Lessons Review Checklist
Please tick all items you've confirmed:
- [ ] Learning outcomes/objectives are clear.
- [ ] Technical concepts introduced in the content are accurate.
- [x] The speed of demonstrations are easy to follow.
- [ ] The narration audio matches what is shown visually.
- [ ] Spelling and grammar are correct.
- [x] Sound quality is consistent throughout the video.
- [x] [Brand Usage Guidelines](https://make.wordpress.org/training/handbook/guidelines/brand-usage-guidelines/) and [Promotional Guidelines](https://make.wordpress.org/training/handbook/guidelines/promotional-guidelines/) are being followed.
- [x] Media assets are all in the public domain ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)).

Leave an additional comment below with feedback. You can also share what you liked about this Tutorial.

--

Notes on Troubleshooting WordPress video tutorial

:35 - “Leaning Outcomes” should be “Learning Outcomes"
1:38 - the ’troubleshooting methods’ are numbered in the voiceover, but not on the screen
- I would consider especially the first few numbers of approaching the issue methodically and downloading a backup to be steps to troubleshoot, not as opposed to separate methods to try interchangeably
- This numbering system is also quickly abandoned, and not returned to. There is another, separate numbered list later
1:53 - "systematically" is pronounced like "systemically" by the voice over
2:27 - how do you see the error logs after adding the Debug lines to the wp-config file? This feels like the first steps in a longer process, and it's not clear how to use this as a troubleshooting step. 
3:14 - what are the next steps if renaming the theme file worked? Even a note saying that 'this means the theme is contributing to the issue, check for an update or revert the theme or choose a resign yourself to your fate and choose a new theme' will help
3:36 - plugin rollback-  I'm not sure how this helps in context. Presumably if they're at this stage, the rollback has not helped to resolve/avert a conflict
4:10 - It sounds like the VO is recommending renaming a singular collective file to deactivate all plugins, not to rename each plugin file individually to deactivate specifically that plugin. The video showed a single plugin file being renamed. 
5:29 - Another set of numbered steps in VO, also not reflected on screen
6:44 -  “white screen of death, plugin or theme conflict…” it feels like the list should keep going, or have an "or" between the two options

General thoughts: 
- This is a great video, this will be a strong resource for users who need to troubleshoot their own sites. 
- I feel like a great distinction could be drawn between 'troubleshooting methods' as in different ways that a user can try to troubleshoot their site, and 'troubleshooting steps' as in the distinct steps users should use while troubleshooting, either in general or as part of one of the specific methods. 
- If there is a numbered list, it would be helpful to see the list on screen, including the previous steps that were part of the same list
- WSOD is presented as the only concrete example of an error, and while plugin and theme conflicts are listed, they are presented in the video as possible causes of the WSOD. If another error such as a Critical Error screen presented, or even a line such as 'the WSOD and other issues', this would help clarify that these methods are all-inclusive.

**Comment by westnz** [comment]
Thank you @cat-og ⭐

**Comment by westnz** [comment]
A few extra observations:
1. I think we can remove 0:27 - 0:57 as it is already covered in the intro
2. I feel we need to use less text-heavy slides and more graphics
4. The voice-over tempo could be slightly faster to maintain a more engaging pace throughout the video.
5. I agree with the comments about the error log section. I think folks should also be shown how to access the wp-config.php file.
6. 2:52 I think folks should also be shown how to get here.
7. 3:16 Ervan, can you make the image static after the first time the arrow moves around? It should not repeat.

**Comment by kspilarski** [comment]
Hi folks, thank you all for the feedback. Great tips and suggestions all around. I'll rework on the script and share it here. 

I was thinking of having the 'Tutorial/Lessons Review Checklist' after the script is submitted. This way, any corrections and edits would be made directly on the script, which would then be reflected in the voiceover and recordings. Then only after that, the video would be created, followed by a separate review checklist specifically for the video. Currently, we have to go back to the beginning to fix the script, which is totally fine since it’s part of the learning and growth process, and we’re aiming for great lessons anyway.

However, @westnz what do you think about having two review stages: one after the script is done, and another after the video, voiceover, and script are all completed? Would it be helpful to add the review processes in the 'Lesson Development Checklist'?

**Comment by westnz** [comment]
Hey Kel, we’ve updated the checklist to make it more concise. Some people prefer to have the script reviewed before recording, so that’s definitely an option if you’d like to do the same.

**Comment by kspilarski** [comment]
Hi @westnz 👋  [here's the edited script](https://docs.google.com/document/d/1aYq6OBRFKQcxbo6g0cCytdZDi7cxOf5FTu5Aae0rR38/edit?usp=sharing), please take a look and share any feedback for improvement. Thank you! If the script is ok, let me know and I'll work on the screen recordings.

**Comment by westnz** [comment]
Hey @mkastler & @cat-og
Could you also please review the updated script before we start re-recording?  Thank you in advance 🙏  

Please note, we have updated the title to make the lesson focused on plugin and theme conflicts.

**Comment by kspilarski** [comment]
Hello @westnz and @ervanyuff I've added [new audio and screen recordings to this folder](https://drive.google.com/drive/folders/1IyHYN_UJ4rWNfhsGSsIv8qSNGfzwWgmi?usp=sharing). 

Also big thanks to @mt8 for sharing valuable FTP tips and creating the error plugin, which made a big difference in getting the new screen recordings ready!

**Comment by westnz** [comment]
That's fantastic. Thank you very much @kspilarski! ⭐ 
@ervanyuff, Can you please go ahead and re-create the video? You will notice there are significant changes. 🙏

**Comment by ervanyuff** [comment]
Hi team, here's [the updated video](https://drive.google.com/file/d/12EdCa344mFpnPn21aG944yAZ2Ztl3xte/view?usp=sharing). Please let me know if there's anything I've missed 🙏

**Comment by westnz** [comment]
Hey @kspilarski 
Can you please re-record the following sentences:
- 2:34 "Or create a manual setup by copying your WordPress files and database to a local site using Studio, DevKinsta or WordPress Playground.
- 3:07 "If you’re comfortable with file editing, access your wp-config.php file."

@ervanyuff 
- Can you please update the thumbnail with the one attached?
- 6:24 - 6:46 The arrow spins too long. Can you please make the spinning arrow stop at 6:33/34 and break up the last 12 seconds using other graphics / screenshots
- 9:40 - 9:42 Remove the 'Conclusion' section![Image](https://github.com/user-attachments/assets/1a49fd9f-fb70-4b3d-b8f8-e09824a1d235)

Thanks, team. Hoping to publish by the end of this week.

**Comment by kspilarski** [comment]
Hi @ervanyuff The new voice recordings with the filenames 2-34 and 3-07 are saved in the Google Drive folder. Thanks!

**Comment by ervanyuff** [comment]
Hi team, here's the [troubleshooting video](https://drive.google.com/file/d/1rt4J0oMbpwwBu8j351TsuLIUShyCmWFR/view?usp=sharing). Please let me know if there's anything I've missed :pray:

**Comment by westnz** [comment]
Thank you both!
Looks good, Ervan.

I will publish this lesson later this week.

**Comment by westnz** [comment]
//published
